src/couch/src/couch_server.erl

```diff
@@ -223,7 +223,7 @@ init([]) ->
     config:enable_feature('pluggable-storage-engines'),
 
     % Mark partitioned databases as a supported feature
-    config:enable_feature(partitions),
+    config:enable_feature(partitioned),
```

Comment:
does it mean that `partitioned` is enabled as default or disabled?

Reply:
It is says is that Partitioned feature is supported. You would need to use the feature_flag to turn it on. In CouchDB the feature_flag is on by default so partitioned databases are supported.